versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-iperf-application.mdx

```diff
@@ -128,17 +135,18 @@ Perform these steps:
 ```
 kubectx <cluster name>
 ```
-2. Create the iPerf namespace using the following command:
+2. Create the `iperf` namespace using the following command:
 ```
 kubectl create ns iperf
 ```
-3. Apply the iperf-sleep.yaml deployment using the following command:
+3. Apply the `iperf-sleep.yaml` deployment using the following command:
 ```
 kubectl apply -f iperf-sleep.yaml -n iperf
 ```
 
 ## Creating the iPerf Server YAML File
-Using the template below, create a deployment file name iperf-server.yaml. All fields in the template will remain the same except for **one <slice name\>** instances which must be replaced with the name of your slice.
+Using the template below, create a deployment file name iperf-server.yaml. All fields in the template will 
+remain the same except for **one <slice name\>** instances which must be replaced with the name of your slice.
```

Comment:
remove the back-slash in <slice name\>

Reply:
"same as above"
![image](https://user-images.githubusercontent.com/105046156/180400178-5e33591e-a93d-4a3b-8eba-c32bb0644cb9.png)

Reply:
try adding backticks as in `<slice  configuration>`